packages/headless/doc-parser/src/api-finder.ts

```diff
@@ -15,5 +15,5 @@ function extractCanonicalName(apiName: string) {
   const dollarIndex = apiName.indexOf('$');
   const endIndex = dollarIndex !== -1 ? dollarIndex : apiName.length;
 
-  return apiName.slice(0, endIndex);
+  return apiName.slice(0, endIndex).trim();
```

Comment:
For some reason, it tried to find the interface `"Controller "` with a space.

Reply:
What is the original `apiName` passed into the function when this happens? Is it `Controller ` (with space)?

Reply:
I just tested and yes, strangely.